mlir/lib/Transforms/RemoveDeadValues.cpp

```diff
@@ -115,6 +157,21 @@ static BitVector markLives(ValueRange values, RunLivenessAnalysis &la) {
   return lives;
 }
 
+// DeletionSet is used to track the Values that are scheduled for removal
```

Comment:
Please document what `updateDeletionSet` does rather than what `deletionSet` variable is for. 

Importantly, please use descriptive function names that document what the corresponding method does. Currently, the function name encodes one of its arguments names - what if we decide to change that later?

Reply:
documented and function renamed, thank you!